pTk/mTk/xlib/X11/X.h

```diff
@@ -179,13 +181,20 @@ are reserved in the protocol for errors and replies. */
 
 #define ShiftMask		(1<<0)
 #define LockMask		(1<<1)
-#define ControlMask		(1<<2)
+#ifndef _WIN32
+#   define ControlMask		(1<<2) /* See bug [9e31fd9449] and below */
+#endif
 #define Mod1Mask		(1<<3)
 #define Mod2Mask		(1<<4)
 #define Mod3Mask		(1<<5)
 #define Mod4Mask		(1<<6)
 #define Mod5Mask		(1<<7)
 
+/* See bug [9e31fd9449], this way prevents conflicts with Win32 headers */
+#ifdef _WIN32
+enum _Bug9e31fd9449 { None = 0, ControlMask = (1<<2) };
```

Comment:
`_Bug9e31fd9449` is a reserved identifier. Why give the enum a name at all?

Reply:
I would direct this concern to upstream Tcl/Tk, see https://core.tcl-lang.org/tk/info/1c618857968b

I do not know if using an enum with a reserved name was intentional, but that is what was used in the suggestion dated 2018-12-21 on https://core.tcl-lang.org/tk/info/9e31fd9449

Reply:
Upstream Tcl/Tk has agreed to not name the enum.